Lombiq.Tests.UI.Samples/Tests/MonkeyTests.cs

```diff
@@ -3,7 +3,9 @@
 using Lombiq.Tests.UI.MonkeyTesting;
 using Lombiq.Tests.UI.MonkeyTesting.UrlFilters;
 using Lombiq.Tests.UI.Services;
+using Shouldly;
```

Comment:
Changes outside this file and `LoggingWebDriverExtensions` are coming from `issue/OSOE-64`.